tests/data/codelist/tier_attribute/valid/tag_sector.yaml

```
@@ -0,0 +1,6 @@
+- Sector:
+    - Agriculture:
+        description: agriculture sector including fishing
+        tier: ^1
+    - Industry:
+        description: industrial sector excluding non-energy use (e.g.feedstocks)
```

For better intuition, items on the same list should (generally) have the same tier - it makes more sense that different hierarchy-levels have different tiers.
```suggestion
    - Industry:
        description: industrial sector excluding non-energy use (e.g.feedstocks)
    - Industry|Chemicals:
        description: chemicals industry
        tier: ^1

```